sympy/parsing/sym_expr.py

```diff
@@ -5,281 +5,275 @@
 lfortran = import_module('lfortran')
 cin = import_module('clang.cindex', import_kwargs = {'fromlist': ['cindex']})
 
-if not lfortran and not cin:
-    class SymPyExpression(object):
-        def __init__(self, *args, **kwargs):
-            raise ImportError('Module not available.')
+if lfortran:
+    from sympy.parsing.fortran.fortran_parser import src_to_sympy
+if cin:
+    from sympy.parsing.c.c_parser import parse_c
+
+@doctest_depends_on(modules=['lfortran', 'clang.cindex'])
+class SymPyExpression(object):  # type: ignore
```

Comment:
Is the `# type: ignore` still needed? I think it was just there because of the duplicate class